README.md

```diff
@@ -13,9 +13,14 @@ $ cp .env.sample .env
 Use the script to create your zip archive:
 
 ```
-$ npm run zip
+$ grunt run
```

Comment:
Good remark

see 314d710 for the documentation.

I've also made a grunt task `grunt package` to just package the zip file.